doc/de/Quick-Start-guide.md

```diff
@@ -1,17 +1,24 @@
 Erste Schritte... 
-==========
+=================
 
 * [Zur Startseite der Hilfe](help)
 
-Das Erste zum Anfang: geh sicher, dass du schon eingeloggt bist. Wenn du noch nicht eingeloggt bist, kannst du das in dem Fenster unten machen. 
+Als Erstes: Gehe sicher, dass Du eingeloggt bist. Wenn Du noch nicht eingeloggt bist, kannst Du das in dem Fenster unten machen. 
 
-Sobald du eingeloggt bist (oder wenn du bereits eingeloggt bist), kannst du unten nun auf deine Profilseite schauen.
+Sobald dies geschehen ist, schaust Du auf Deine Profilseite (https://***************.tls/profile/*****).
 
```

Comment:
But I think that this sentence isn't correct at all because when I log in I am on the network page.
Shouldn't it be changed to "Sobald dies geschehen ist, klicke auf den Reiter 'Pinnwand'. "?